docs/devices-api/more-info/models.mdx

```diff
@@ -340,7 +340,9 @@ export interface LatLon {
 ```typescript
 export interface Metadata {
   date: string;
+  hour?: string;
```

Comment:
We'll need to update all of our docs to include this as an optional field with a description of expected format - right?